website/source/docs/runtime/environment.html.md

```diff
@@ -61,6 +61,14 @@ environment variables.
     <td>The job's name</td>
   </tr>
   <tr>
+    <td>`NOMAD_DC`</td>
+    <td>The datacenter in which the job is running.</td>
```

Comment:
Pedantic, but these aren't sentences, so we should not have a period at the end 😄